src/Components/WebView/Samples/PhotinoPlatform/testassets/PhotinoTestApp/Program.cs

```diff
@@ -25,7 +25,9 @@ static void Main(string[] args)
 
         AppDomain.CurrentDomain.UnhandledException += (sender, error) =>
         {
-            mainWindow.Photino.OpenAlertWindow("Fatal exception", error.ExceptionObject.ToString());
+            Console.Write(
```

Comment:
The old code doesn't work in Photino anymore (apparently it was never supposed to work).